app/controllers/content_files_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+# Controller for a Work content file
+class ContentFilesController < ApplicationController
+  # Skipping authorization for now
```

Comment:
Could you prepend a `TODO: ` here and/or create a ticket if we don't have one?

Reply:
https://github.com/sul-dlss/hungry-hungry-hippo/issues/152